doc/reference_coverage.xml

```diff
@@ -0,0 +1,195 @@
+      <para>For invalid coverage inputs, non-shared edges are ignored, and only shared edges are simplified.</para>
+      <para>To only simplify the "internal" edges of the set of polygons (those that are shared by two polytons) set the "simplifyBoundary" parameter to 1.</para>
+      <note><para>You must ensure that your coverage is valid before running simplification or you may get unexpected results in the output, such as boundary intersections or non-shared boundaries where you thought there were shared boundaries.</para></note>
+      <para>Availability: 3.4.0 - requires GEOS &gt;= 3.12.0</para>
+
+    </refsection>
+
+    <refsection>
+      <title>Examples</title>
+
+      <programlisting>CREATE TABLE coverage (id integer, geom geometry);
+
+INSERT INTO coverage VALUES
+  (1, 'POLYGON((0 0, 10 0, 10.1 5, 10 10, 0 10, 0 0))'),
+  (2, 'POLYGON((10 0, 20 0, 20 10, 10 10, 10.1 5, 10 0))'),
+  (3, 'POLYGON((20 0, 30 0, 30 10, 20 10, 20 0))');
+
+SELECT id, ST_AsText(ST_CoverageSimplify(geom, 1.0) OVER ())
+  FROM coverage;
+
+ id |               st_astext
+----+---------------------------------------
+  1 | POLYGON((10 0,10 10,0 10,0 0,10 0))
+  2 | POLYGON((10 0,20 0,20 10,10 10,10 0))
+  3 | POLYGON((20 0,30 0,30 10,20 10,20 0))
+      </programlisting>
+    </refsection>
+
+    <refsection>
+      <title>See Also</title>
+      <para>
+        <xref linkend="ST_CoverageUnion" />,
+        <xref linkend="ST_CoverageInvalidLocations" />
+      </para>
+    </refsection>
+
+  </refentry>
+
+  <refentry id="ST_CoverageUnion">
+    <refnamediv>
+      <refname>ST_CoverageUnion</refname>
+
+      <refpurpose>Simplifies the edges of an implicit coverage formed by a set of polygons, applying the Visvalingam–Whyatt algorithm to the edges.</refpurpose>
+    </refnamediv>
+
+    <refsynopsisdiv>
+      <funcsynopsis>
+        <funcprototype>
+          <funcdef>geometry <function>ST_CoverageUnion</function></funcdef>
+          <paramdef><type>geometry set</type>
+            <parameter>geom</parameter></paramdef>
+        </funcprototype>
+      </funcsynopsis>
+    </refsynopsisdiv>
+
+    <refsection>
+      <title>Description</title>
+
+      <para>When the input are a polygonal coverage, the function will provide a high speed merge of adjacent polygons in the aggregation set. This function only works with inputs that are coverages: no overlaps, all shared edges identical down to the vertex level. See <xref linkend="ST_CoverageInvalidLocations" /> for a way to test that a set of polygons are a valid polygonal coverage.</para>
+
+      <note><para>You must ensure that your coverage is valid before running simplification or you may get unexpected results in the output, such as unmerged inputs returning as multipolygons.</para></note>
+
+      <para>Availability: 3.4.0 - requires GEOS &gt;= 3.12.0</para>
```

Comment:
This is available from GEOS 3.8, I think.

Reply:
Not backported, but yes: https://github.com/libgeos/geos/blob/main/NEWS.md#changes-in-380

Reply:
If I make it actually available as of GEOS 3.8 I need a whole special set of ifdefs just for coverage union. I figured on just introducing all coverage functions as of a GEOS 3.12 ifdef. Is that alright?

Reply:
Don't you already have it ifdeffed on 3.8? https://github.com/postgis/postgis/pull/731/files#diff-cd95a4eccec3e850213b9d8ec7fd08a2365b44458f1e71f65aa73bbb799dc3d0R798